artichoke-backend/src/class/registry.rs

```diff
@@ -225,4 +225,13 @@ where
     pub fn shrink_to_fit(&mut self) {
         self.0.shrink_to_fit();
     }
+
+    /// Shrinks the capacity of the symbol table with a lower bound.
```

Comment:
whoops, it looks like this docstring should refer to "registry" instead of "symbol table".